### PR TITLE
wire clusterapi v28

### PR DESCRIPTION
**Pull request by mazzy89**

### CODE REVIEW

File: service/controller/clusterapi/drainer.go [hunk 1]
```diff
@@ -16,7 +16,8 @@ import (
 	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
 
 	awsclient "github.com/giantswarm/aws-operator/client/aws"
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27"
+	v27 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v27"
+	v28 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v28"
```
Comment (xh3b4sd) [inline service/controller/clusterapi/drainer.go]: Please don't. Further, we can drop v27 when we have v28.

Comment (mazzy89) [reply service/controller/clusterapi/drainer.go]: Sorry. I did not do on purpose. VS autocompletion

**Comment by mazzy89** [comment]
@xh3b4sd can you take a look to check if I drop out `v27` correctly? thank you

File: new-version-bundle [hunk 2]
```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+set -eu
```
Comment (xh3b4sd) [inline new-version-bundle]: What is this script?

Comment (mazzy89) [reply new-version-bundle]: ouch :-( it was the script to automate the new version bundle. going to remove it

**Pull request merged**